src/deno_json/mod.rs

```diff
@@ -192,6 +395,97 @@ impl FmtOptionsConfig {
       && self.semi_colons.is_none()
+      && self.quote_props.is_none()
+      && self.new_line_kind.is_none()
+      && self.use_braces.is_none()
+      && self.for_in_statement_use_braces.is_none()
+      && self.for_of_statement_use_braces.is_none()
+      && self.for_statement_use_braces.is_none()
+      && self.if_statement_use_braces.is_none()
+      && self.while_statement_use_braces.is_none()
+      && self.brace_position.is_none()
+      && self.arrow_function_brace_position.is_none()
+      && self.class_declaration_brace_position.is_none()
+      && self.class_expression_brace_position.is_none()
+      && self.constructor_brace_position.is_none()
+      && self.do_while_statement_brace_position.is_none()
+      && self.enum_declaration_brace_position.is_none()
+      && self.for_in_statement_brace_position.is_none()
+      && self.for_of_statement_brace_position.is_none()
+      && self.for_statement_brace_position.is_none()
+      && self.function_declaration_brace_position.is_none()
+      && self.function_expression_brace_position.is_none()
+      && self.get_accessor_brace_position.is_none()
+      && self.if_statement_brace_position.is_none()
+      && self.interface_declaration_brace_position.is_none()
+      && self.module_declaration_brace_position.is_none()
+      && self.method_brace_position.is_none()
+      && self.set_accessor_brace_position.is_none()
+      && self.static_block_brace_position.is_none()
+      && self.switch_case_brace_position.is_none()
+      && self.switch_statement_brace_position.is_none()
+      && self.try_statement_brace_position.is_none()
+      && self.while_statement_brace_position.is_none()
+      && self.single_body_position.is_none()
+      && self.for_in_statement_single_body_position.is_none()
+      && self.for_of_statement_single_body_position.is_none()
+      && self.for_statement_single_body_position.is_none()
+      && self.if_statement_single_body_position.is_none()
+      && self.while_statement_single_body_position.is_none()
+      && self.next_control_flow_position.is_none()
+      && self.do_while_statement_next_control_flow_position.is_none()
+      && self.if_statement_next_control_flow_position.is_none()
+      && self.try_statement_next_control_flow_position.is_none()
+      && self.trailing_commas.is_none()
+      && self.arguments_trailing_commas.is_none()
+      && self.parameters_trailing_commas.is_none()
+      && self.array_expression_trailing_commas.is_none()
+      && self.array_pattern_trailing_commas.is_none()
+      && self.enum_declaration_trailing_commas.is_none()
+      && self.export_declaration_trailing_commas.is_none()
+      && self.import_declaration_trailing_commas.is_none()
+      && self.object_expression_trailing_commas.is_none()
+      && self.object_pattern_trailing_commas.is_none()
+      && self.tuple_type_trailing_commas.is_none()
+      && self.type_literal_trailing_commas.is_none()
+      && self.type_parameters_trailing_commas.is_none()
```

Comment:
Not sure if it's a good idea to be this granular. Having one trailing commas option like other formatters do seems more than enough.

Reply:
Yeah, I don't think we should expose the AST specific config.

Reply:
Sounds fair. Updated to not include AST specific rules.